### DISCUSSION TITLE
Questions about Bootstrap 4 to 5 upgrade

### DISCUSSION THREAD

**Discussion by techboyg5** [twbs/bootstrap#31034] [General] [answered]
I have a few questions about the upgrade from Bootstrap 4 to 5:
1. Will there be an upcoming 4.5.1 or 4.6.0 release?
2. What changes would I have to make to my code to upgrade to v5? I have 40 pages using Bootstrap 4.5.0.
3. Will v4 still be supported after v5 comes out? If yes, then how long?

**Comment by mdo** [comment, marked as answer]
1. v4.5.1 is being worked on, but we're prioritizing v5 alpha first. Unsure about a v4.6.0 at this point—depends on if we add more features.
2. Tons, but not as much as the move from v3 to v4. We'll have the migration details in our v5 alpha docs. Bare in mind v5 is still only going to be in alpha, so migrating isn't necessarily advised at this point.
3. Yes, you can read more about our general support for versions at https://github.com/twbs/release. I imagine there will be several months still of v4 support.
